### ISSUE TITLE
[Suggestion] Database for user-created profiles

### ISSUE THREAD

**Issue by AkiraJkr** [idolize/xcloud-keyboard-mouse#24]
That way, users can upload their own creations to make easy setups for other users, specially for shooter games like Halo Infinite.

The way that the database can be hosted, if GitHub is used, could be like this:

 - Profile Database
     - Games
       - Halo Infinite
       - Forza Horizon 5
       - ETC
     - Universal (Though honestly? I don't see where this would be necessary, specially for shooters)
       - Shooter
       - Racing
       - Platformer

Meanwhile, in the extension itself, there could be a Upload to Database button with info like:

Current Game:
Profile Name:
Profile Creator Comments: (With perhaps, some encouragement to include commentary like; What in-game settings were used, buttons to confirm/go back...

**Comment by idolize** [comment]
Hey @AkiraJkr - cool idea! Don't most people use custom key bindings for games though? Is the goal here to match the original keybindings for M+KB for each of these games?

**Comment by AkiraJkr** [comment]
Yep. That would be correct. Or maybe someone figured out a magical configuration that works perfectly with a specific game, and wants to share it with others.
